facilitator/src/task.rs

```diff
@@ -66,6 +66,8 @@ pub trait Task: Debug + Display + Sized + serde::de::DeserializeOwned {}
 #[derive(Debug, Deserialize, PartialEq)]
 #[serde(rename_all = "kebab-case")]
 pub struct IntakeBatchTask {
+    /// The trace identifier for the intake, typically a UUID
+    pub trace_id: Option<String>,
```

Comment:
I was trying to figure out why this has to be `Option<String>`, and the explanation I came up with is that when we deploy a `workflow-manager`/`facilitator` that support trace IDs, there will still be queued up tasks that don't have trace IDs, so `facilitator` needs to handle both. If that's why this is `Option`, could you please (1) explain that in a comment and (2) file an issue to remove support for tasks without trace IDs once we are sure we don't need to support them anymore?

Reply:
Yep! That was the reason I made them optional. I will add the explanation and the issue!